doc/boot.md

```diff
@@ -158,7 +158,6 @@ same upgrade command again will bring both partitions into sync.
 
 [RAUC]: https://rauc.io
 
-
```

Comment:
Style thing, I try to keep double linefeeds between sections.

Reply:
Oops, thx!